_posts/2022-04-28-tiaas-hans-rudolf-lucille.md

```diff
@@ -0,0 +1,18 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Training Infrastructure Feedback from Hans-Rudolf Hotz and Lucille Delisle'
+tags: [training, TIaaS]
+supporters:
+- galaxy-europe
+author: [hrhotz, lldelisle]
```

Comment:
not sure if this will work. if this does not map to `peoples` it should maybe be a full name?

Reply:
can only authors in the people.yml file be added with their IDs?

Reply:
I think so, not sure.